core-services/release-controller/_releases/release-ocp-4.5-ci.json

```diff
@@ -113,7 +113,7 @@
       "prowJob":{"name":"release-openshift-origin-installer-e2e-azure-ovn-upgrade-4.4-stable-to-4.5-ci"}
     },
     "upgrade-ovirt-minor":{
-      "cron":"@every 48h",
+      "cron":"0 9 * * *",
```

Comment:
This job went from running once every 48 hours to running once a day?

Reply:
Yes I want to get a big more results, and I belive if it will not run each in parallel to other jobs its ok